cfpb/rental-assistance-finder
# Update Virgin Islands entry in states.json

The entry for Virgin Islands lacked the "s" and wasn't finding results in the data file.

## Changes

- Add "s" to Virgin Islands in states.json

## Testing

1. Check out branch
2. Select "Virgin Islands" in dropdown and verify that there is a result
